### PR TITLE
Clarify variable names for payment channels in `#send_payment`

### PR DESCRIPTION
**Pull request by ramontayag**
To lessen confusion with the terms `account` and `source_account`, rename the payment channel option to `channel_account`. Since the gem hasn't been released yet, there's no breaking change.

New signature:

```ruby
client.send_payment(
  from: from_account,
  to: destination_account,
  amount: Stellar::Amount.new(0.55),
  channel_account: channel_account,
 )
```

### CODE REVIEW

**Comment by ramontayag** [comment]
@bartekn mind taking a look?

File: lib/stellar/client.rb [hunk 1]
```diff
@@ -108,25 +109,33 @@ def create_account(options={})
       amount:   Stellar::Amount
     }) => Any
     def send_payment(options={})
-      from     = options[:from]
-      sequence = options[:sequence] || (account_info(from).sequence.to_i + 1)
+      from_account     = options[:from]
+      channel_account = options[:channel_account]
```
Comment (bartekn) [inline lib/stellar/client.rb]: Can we change `channel_account` to, simply, `transaction_source`. I think that name is less confusing.

**Comment by ramontayag** [comment]
@bartekn renamed to `transaction_source` :+1:

File: lib/stellar/client.rb [hunk 2]
```diff
@@ -108,25 +109,29 @@ def create_account(options={})
       amount:   Stellar::Amount
     }) => Any
     def send_payment(options={})
-      from     = options[:from]
-      sequence = options[:sequence] || (account_info(from).sequence.to_i + 1)
+      from_account     = options[:from]
+      tx_source_account = options[:transaction_source] || from_account
+      op_source_account = from_account if tx_source_account.present?
```
Comment (bartekn) [inline lib/stellar/client.rb:114]: `tx_source_account` will always be `present`, and equal `options[:from]` or `options[:transaction_source]` right? It's important as it's used later in line 128.

Comment (ramontayag) [reply lib/stellar/client.rb:114]: Yes, the tx source will be `:transaction_source` if present, then fallback to `:from`.

**Comment by ramontayag** [comment]
Hi @bartekn just following up on this :)

**Review by bartekn — APPROVED** [review]

**Pull request merged**